pootle/settings/90-dev-local.conf.sample

```diff
@@ -248,3 +248,6 @@ MT_BACKENDS = [
 
 # URL for the amaGama TM server.
 AMAGAMA_URL = 'https://amagama-live.translatehouse.org/api/v1/'
+
+# Silence the DEBUG check on dev servers
+SILENCED_SYSTEM_CHECKS = ["pootle.W005"]
```

Comment:
As I mentioned when I proposed this, I think the real value is silencing this by default, i.e. in any other conf file and not this sample file.